solr/core/src/test/org/apache/solr/cloud/AtomicUpdateBlockShardedTest.java

```diff
@@ -130,7 +130,7 @@ public void doNestedInplaceUpdateTest() throws Exception {
     // for now,  we know how ranges will be distributed to shards.
     // may have to look it up in clusterstate if that assumption changes.
 
-    SolrInputDocument doc = sdoc("id", "a", "level_s", "root", "_root_", "a");
+    SolrInputDocument doc = sdoc("id", "a", "level_s", "root");
```

Comment:
@dsmiley, this is what I meant by "remove hard coded _root_.
I added these since my logic depended on chidless documents having _root_.
When SOLR-5211 was merged, I just rebased on master and removed these hardcoded _root_ fields, which are now automatically generated by Solr.